src/scripts/install.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/bash
 echo "Running reflector to sort for fastest mirrors"
-flatpak-spawn --host pkexec reflector --latest 5 --sort rate --save /etc/pacman.d/mirrorlist
-flatpak-spawn --host pkexec jade config ~/.config/jade.json
+pkexec reflector --latest 5 --sort rate --save /etc/pacman.d/mirrorl
```

Comment:
Is this meant to be `mirrorlist` rather than `mirrorl`?

Reply:
yep, mirrorlist